docs/content/standards/closed-loop-token.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Closed Loop Token
+---
+
+import BalanceTokenCoin from '@site/static/img/balance-token-coin.png';
+
+Closed Loop Token allows its creator to limit applications where the token is used, set up custom policies for transfers, spending and conversions. It is defined in the `sui::token` module and is a part of the Sui Framework.
+
+## Background & Use Cases
+
+The standard Coin implementation on Sui is an example of an open-loop system - Coins are free-flowing, [wrappable](/concepts/object-ownership/wrapped), [freely transferable](/concepts/dynamic-fields/transfers/custom-rules#the-store-ability-and-transfer-rules) and can be stored in any application. The best real world analogy would be "cash" - hardly regulated and can be freely used and passed.
```

Comment:
More a comment for @ronny-mysten (and I think this is already recorded somewhere), but this reminds me that the `transfer/custom-rules` docs should not be under "dynamic fields".

Reply:
It may be easier to follow if you start this section with the analogy that tokens are like bank accounts, and then follow it up with the example of `Coin` being like cash, because the main topic here is what tokens are, so it can be confusing for people to understand where the story is going if the first paragraph is about what tokens are not.